docs/root/configuration/http/http_filters/stateful_session_filter.rst

```diff
@@ -0,0 +1,80 @@
+.. _config_http_filters_stateful_session:
```

Comment:
I wonder if we should include a note in this documentation that this type of sticky session is an anti-pattern and that operators should avoid using this feature when possible?

Reply:
I think this function is not that special, and there are similar functions among other proxies. But I can add a note to let users use this feature with caution. 

Similar function in other proxy: https://docs.nginx.com/nginx/admin-guide/load-balancer/http-load-balancer/#enabling-session-persistence

Reply:
Let's add it. Something like:

```
.. note::

 Stateful sessions can result in imbalanced load across upstreams and allow external actors to direct requests to
 specific upstream hosts. Operators should carefully consider the security and reliability implications of stateful 
 sessions before enabling this feature.
```